File: converter/app/js/converter.js
async function convert_js()
{
    let original = document.getElementById('path').value.replace(/^.*\\/,"");
    let format = document.getElementById('selection').value;
    let file = document.getElementById('get_from_file').value.replace(/^.*\\/,"");
    let filename = document.getElementById('result').innerHTML = file

    if(!file && !original)
    {
        return alert("Вы не выбрали файл ! Либо поле пустое !")
    }
    else if(!original)
    {
        filename
        await eel.convert(file, format)();
        alert("Конвертация прошла успешно !")
        return window.location.reload();
    }
    else
    {
        await eel.convert(original, format)();
        alert("Конвертация прошла успешно !")
        return window.location.reload()
    }
}

File: converter/converter.py
from eel import init, start, expose
from os import path, environ
import aspose.words as aspose

init('app')

@expose
def convert(original, converted):
    document = aspose.Document(original)
    desktop = path.join(path.join(environ['USERPROFILE']), 'Desktop')
    result = path.join(desktop, f"{path.splitext(original)[0]}.{converted}")
    document.save(result)

start('main_eng.htm', size = (576, 920))